opm/grid/polyhedralgrid/grid.hh

```diff
@@ -848,61 +848,84 @@ namespace Dune
       ijk[2] = gc / logicalCartesianSize()[1];
     }
 
+    /// \name Parallel grid extensions.
+    /// Methods extending the DUNE's parallel grid interface.
+    /// These are basically for scattering/gathering data to/from
+    /// distributed views.
+    //@{
+    ///
+    /// \brief Moves data from the global (all data on process) view to the distributed view.
+    ///
+    /// This method does not do communication but assumes that the global grid
+    /// is present on every process and simply copies data to the distributed view.
+    /// \tparam DataHandle The type of the data handle describing the data and responsible for
+    ///         gathering and scattering the data.
+    /// \param handle The data handle describing the data and responsible for
+    ///         gathering and scattering the data.
+    template<class DataHandle>
+    void scatterData(DataHandle& handle) const
```

Comment:
I am a bit torn here.
Providing a function that does not do what it says it would might be a bit surprising. If we keep it, then we should at least unconditionally throw an exception, if there are more than 1 MPI processes. Or work around it at the call site.

Reply:
e.g. by specializing for CpGrid or using the vanguards. Where is it called BTW?

Reply:
The scatterData and the communicate method are just copied from the CPGRID. MPI does not work with polyhedrealgrid so I will make sure the methods throw if they are called and cleanup the comments.